test/setup_bats_and_rbenv_and_run_tests

```diff
@@ -4,7 +4,9 @@ BATS_TAG=v0.4.0
 RBENV_TAG=v0.4.0
 
 git clone --depth 1 --branch $BATS_TAG https://github.com/sstephenson/bats.git
+PATH="$PWD/bats/bin:$PATH"
+
 git clone --depth 1 --branch $RBENV_TAG https://github.com/sstephenson/rbenv.git
+PATH="$PWD/rbenv/bin:$PATH"
```

Comment:
ideally, the test suite would be using stubs for rbenv behavior so rbenv itself wouldn't be necessary. Then this entire script could be killed and instead handled in .travis.yml:

```
install: git clone --depth 1 https://github.com/sstephenson/bats.git
script: ./bats/bin/bats test
```